util/dasharo/tag-with-changelog

```diff
@@ -166,7 +166,7 @@ get_log_submodule() {
 # up manually.
 version_ctrl_c() {
 	printf "\n Cleaning up and exiting.\n" >&2
-	git checkout origin/main > /dev/null 2>&1
+	git checkout $NEW_GIT_VERSION > /dev/null 2>&1
```

Comment:
Then can should also be quoted.

Reply:
https://github.com/Dasharo/coreboot/compare/b20dcc26401c9ec1656c191cc2d9e1c47ab0f743..97105c6159c8ca5a10eccf425310b9f4339ad377